miden/tests/integration/stdlib/sys.rs

```diff
@@ -0,0 +1,34 @@
+use super::build_test;
+use proptest::prelude::*;
+use rand_utils::rand_vector;
+use vm_core::MIN_STACK_DEPTH;
+
+#[test]
+fn stack_cleanup() {
+    let source = "use.std::sys begin push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 push.0 exec.sys::stack_cleanup end";
```

Comment:
nit: you can probably replace a bunch of `push.0` operations with `repeat.25 push.0 end`.